src/components/trace/hat-script-graph.ts

```diff
@@ -115,6 +115,10 @@ export class HatScriptGraph extends LitElement {
   };
 
   private render_action_node(node: Action, path: string, graphStart = false) {
+    if (node === null) {
```

Comment:
Actually thinking about it, instead of filtering it out here, we should guard for it and not call this function when it's `null` instead.

Reply:
I thought about that as well, but then we have to duplicate the check a few times, since this function gets called in multiple places, so I went for the central approach (also prevents having to account for that scenario in case of future additions to the graph logic). But if preferred, I can move the check out to the different calls.

Reply:
We should move it to the different calls, because if we ever have a bug where `null` is injected somewhere, it would now be silently ignored. We want to be explicit where we accept `null`.